### DISCUSSION TITLE
Welcome to Flutter Any Logo Discussions!

### DISCUSSION THREAD

**Discussion by JordyHers** [JordyHers/flutter_any_logo#43] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EmmanuelQC** [comment]
Hello Jordy!
Awesome package!

I'm an incoming computer science student working on some side projects to help with my applications to different universities.

I came across this package when a friend recommended it to me. 

I just have a suggestion - 
Is it possible to access the logos with a string in a function?
something like: AnyLogo('tech.daily.apple').

Let me know if there's any way I can help!

Have a great day.

**Reply by JordyHers** [reply]
@EmmanuelQC Hello there. Thabk you for your message and sorry for the late response. I can definitely dig up and find out out to implement your suggestion. Thank you again for trusting us and using our package.
